src/main/java/baritone/process/MineProcess.java

```diff
@@ -115,7 +111,7 @@ public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
             Baritone.getExecutor().execute(() -> rescan(curr, context));
         }
         if (Baritone.settings().legitMine.value) {
-            addNearby();
+            if (!addNearby()) return null;
```

Comment:
You forgot to cancel here so this crashes the game with an `IllegalStateException` from `PathingControlManager`